mne/tests/test_utils.py

```diff
@@ -104,6 +104,29 @@ def test_object_size():
                     msg='%s < %s < %s:\n%s' % (lower, size, upper, obj))
 
 
+def test_get_inst_data():
+    from mne.epochs import _segment_raw
+    from mne.time_frequency import tfr_morlet
+
+    raw = read_raw_fif(fname_raw)
+    # chns = raw.ch_names[:5]
+    # raw.pick_channels(chns)
```

Comment:
Ah, I didn't notice - in order to make tests fast I wanted to pick channels before loading the data - forgot it is not possible.